### PR TITLE
Add CI tests and check syntax for instrument/vocal

### PR DESCRIPTION
**Pull request by yvanzo**
Because it is an in-house format that is expected to be dropped eventually, this check isn’t part of Weblate source code repository. Therefore it must have continuous integration tests to detect any breakage with Weblate update in the future. This is a requirement by the Weblate team to deploy it as a custom check.

Additionally it detects and checks the specific [syntax for instrument and vocal](https://wiki.musicbrainz.org/MusicBrainz_Server/Internationalization#Variables).

### CODE REVIEW

**Comment by yvanzo** [comment]
The YAML file is a mix from [WeblateOrg/fedora_messaging](https://github.com/WeblateOrg/fedora_messaging/blob/main/.github/workflows/test.yml) and [WeblateOrg/weblate](https://github.com/WeblateOrg/weblate/blob/main/.github/workflows/test.yml) (as there is no such `test.yml` file in [WeblateOrg/customize-example](https://github.com/WeblateOrg/customize-example).)

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        python-version:
+        - '3.9'
+        - '3.10'
+        - '3.11'
+        - '3.12'
```
Comment (mwiencek) [inline .github/workflows/test.yml:18]: Is it necessary to test against all of these older versions?

**Review by reosarevok — APPROVED** [review]
As far as I understand how this works this seems good and sensible to me :)

**Review by mwiencek — APPROVED** [review]
The actual tests look good. I didn't review the workflow, since it successfully runs.

File: .github/workflows/test.yml (hunk 1 above)
Comment (yvanzo) [reply .github/workflows/test.yml:18]: Probably not, but I don’t know which version is powering our instance, so I just sticked to the versions tested in other repositories. See https://github.com/metabrainz/musicbrainz-weblate-checks/pull/2#issuecomment-1802241773

**Pull request merged**